aphrodite/common/sampling_params.py

```diff
@@ -199,7 +216,10 @@ class SamplingParams(
     xtc_threshold: float = 0.1
     xtc_probability: float = 0
     nsigma: float = 0.0
-
+    dry_multiplier: float = 0.0
+    dry_base: float = 1.75
+    dry_allowed_length: int = 2
+    dry_sequence_breaker_ids: List[float] = []
```

Comment:
Is this supposed to be a list of int?
```suggestion
    dry_sequence_breaker_ids: List[int] = []
```